### ISSUE TITLE
Glitchy-looking scrolling on iOS Safari

### ISSUE THREAD

**Issue by nolanlawson** [tootsuite/mastodon#3407]
Not sure if this is a regression that was introduced in 1.4.1 or if it always existed, but scrolling can look kind of jumpy/jerky/glitchy on iOS Safari (10.3.1): https://gfycat.com/TiredAmazingBordercollie

I can't reproduce this issue on mobile Chrome or Firefox on an Android device. :confused:

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by daprice** [comment]
I've experienced this in MacOS Safari as well. The jumpy scrolling in @nolanlawson's recording is something I've noticed since I started using Mastodon several months ago (I attributed it to posts being added to the timeline during scrolling), but as of 1.4 or 1.4.1 it's worse and sometimes it starts jumping back and forth rapidly after the scroll finishes, and I have to reload the page to stop it.

**Comment by daprice** [comment]
Not sure how helpful this is, but I opened the web inspector while it was happening and it looks like the first and last visible toots’ `<div>`s are constantly having a style tag added and removed multiple times a second, causing a feedback loop where they come in and out of the viewable area.

I uploaded a recording including the web inspector (warning: may cause seizures) [here](https://gfycat.com/EntireSingleArcherfish). Note that I wasn’t scrolling during this; it started after I scrolled down and more toots came into view. It’s difficult to replicate as it doesn’t do this after every scroll, but it seems like there are certain toots that always trigger it when they come into view (possibly ones containing images?).
